working-with-openai/setting-up-openai-postman.md

```diff
@@ -0,0 +1,91 @@
+## Setting up the OpenAI API in Postman
+
+## Goals
+Now that we have our OpenAI account set up and verified, let's  get OpenAI set up in Postman and Python so we are prepared to start testing our API calls and bringing them into our project.
+
+Our goals for this lesson are to:
+- Set up an OpenAI environment in Postman
+- Make our first Postman API 
```

Comment:
I'm a little unclear on what is meant here, are we creating a collection in Postman, or making our first call to an OpenAI API (or both)?